NMS-14346: Document SNMP oid varbind pattern matching (#5041)
OpenNMS/opennms: docs/modules/operation/pages/events/event-configuration.adoc

When setting event severities, it's important to consider each event in the context of your infrastructure as a whole.
Events whose severity is critical at the zoomed-in level of a single device may not merit a `Critical` severity in the zoomed-out view of your entire enterprise.

> Since an event with `Critical` severity can never have its alarms escalated, this severity level should usually be reserved for events that unequivocally indicate a truly critical impact to the business.

Author: Ya, I heard. I love Spinal Tap, but it seems out of place. I don't feel strongly about it, though, so if there's going to be rioting, I'm fine keeping it. What do you think?

> Since an event with `Critical` severity can never have its alarms escalated, this severity level should usually be reserved for events that unequivocally indicate a truly critical impact to the business.

Author: Added back in.

`%hardware[<token>]%`::
    The hardware field <token>'s value.

[[parameter-tokens]]
=== Parameter tokens

> Many events carry additional information in *parameters* (see <<events/anatomy-events.adoc#ga-events-anatomy-of-an-event, Anatomy of an Event>>).
> These parameters may start life as SNMP trap *variable bindings*, or *varbinds* for short.

Reviewer: ```suggestion
Many events carry additional information in their parameters (see <<events/anatomy-events.adoc#ga-events-anatomy-of-an-event, Anatomy of an event>>).
These parameters may start life as SNMP trap variable bindings (varbinds).
```
